ui-backend/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/handlers/CqlTransformHandler.java

```diff
@@ -118,22 +133,88 @@ public void setArguments(Map<String, Object> arguments) {
     }
   }
 
+  public class CqlTransformRequest {
+    private List<CqlRequestImpl> searches;
+    private int count;
+    private List<CqlRequest.Sort> sorts;
+    private List<String> hiddenResults;
+
+    public CqlTransformRequest() {
+      this.searches = Collections.emptyList();
+      this.count = 0;
+      this.sorts = Collections.emptyList();
+      this.hiddenResults = Collections.emptyList();
+    }
```

Comment:
✏️ You can initialize these directly and remove this constructor to save on a few lines of code
```
    private List<CqlRequestImpl> searches = Collections.emptyList();
    private int count = 0;
    private List<CqlRequest.Sort> sorts = Collections.emptyList();
    private List<String> hiddenResults = Collections.emptyList();
```